src/index.html

```diff
@@ -1,11 +1,25 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
-</head>
-<body>
-    <h1>Hello webpack!</h1>
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Meals App</title>
+    <link rel="stylesheet" href="style.css">
```

Comment:
Great job applying the styles on the header! however, since we are using webpack for this project and we set the templates in the webpack configuration file, we do not need to link the CSS file with this method anymore.

Instead of this please import the CSS file into the index.js file and serve the app using the webpack server to see the changes to your styles.